src/components/app/header/header.js

```diff
@@ -41,18 +79,40 @@ class Header extends Component {
         </div>
         <div className="label">{ this.props.t('header.appName') }</div>
         <div className="items-container">
+          <div className="menu-container">
```

Comment:
We have other places that will have menus, right? For example, where there will be a menu of actions for device alerts... which currently is just a Link to the alert, but that they want to be a menu containing the link as well as actions to ack or close right there without navigating. I think there are other similar "in grid" menus as well.
Any reason to not go ahead and create a menu component now?

Reply:
In the future we will have menus but I'm not building one right now for this tasks. Rather, I'm reusing what we already have. This may well be a starting point for a menu in the future. To build a full on reusable menu would take more thinking into how to shared common event listeners, etc.